src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -57,16 +77,18 @@ public void adsRecoveryNoAdblockPageFairOasis() {
   )
   @Execute(asUser = User.USER_2)
   public void adsRecoveryLoggedInPageFairOasis() {
-    String url = urlBuilder.getUrlForPath(WIKIA, getUrlArticlePageFairRecovery());
+    String url = urlBuilder.getUrlForPath(WIKIA, getUrlArticlePageFairRecovery(true));
     AdsRecoveryObject adsRecoveryObject = new AdsRecoveryObject(driver, url, DESKTOP_SIZE);
 
     adsRecoveryObject.verifyNumberOfPageFairRecoveredSlots(0);
   }
 
-  private String getUrlArticlePageFairRecovery() {
+  private String getUrlArticlePageFairRecovery(boolean isPAL) {
     String url = urlBuilder.appendQueryStringToURL(WIKIA_ARTICLE, INSTANT_GLOBAL_INSTART_LOGIC_SWITCHED_OFF);
     url = urlBuilder.appendQueryStringToURL(url, INSTANT_GLOBAL_PAGE_FAIR_SWITCHED_ON);
 
-    return urlBuilder.appendQueryStringToURL(url, INSTANT_GLOBAL_PREMIUM_AD_LAYOUT_SWITCHED_OFF);
+    return urlBuilder.appendQueryStringToURL(url, isPAL ?
```

Comment:
Maybe we could make a universal helper method that enables PremiumAdLayout for any URL instead of adding a flag to selected methods?

Reply:
maybe we could make a universal helper that enables or disables *any* wgVar globally?